.python-version

```diff
@@ -0,0 +1,6 @@
+3.9.2
+3.8.8
+3.7.10
+3.6.13
+3.5.10
+2.7.18
```

Comment:
I think this file should be put in .gitignore and removed from this PR, as it may conflict with the local developer's own `.python-version` and it is one more thing to maintain.

Reply:
Done!